### ISSUE TITLE
CPU Frequency Scaling not available for 3.12

### ISSUE THREAD

**Issue by bigeagle** [beagleboard/kernel#80]
Hi,

Recently I updated my BBB kernel to 3.12.3-bone9, but after that, CPU frequency scaling became unavailable.

There's no `cpufreq` in `/sys/devices/system/cpu/cpu0/`, and `cpupower frequency-info` returns:

```
analyzing CPU 0:
  no or unknown cpufreq driver is active on this CPU
```

I'm using default config and didn't modify kernel code, and the installed distribution is archlinuxarm.

cheers,

**Comment by RobertCNelson** [comment]
Correct that's one of the defects of v3.12.x, looking at the v3.13-rcX merge window this functionality should now be in mainline.. So we just need a list of cherry picks.. ;)

**Comment by RobertCNelson** [comment]
This is fixed in both 3.12/3.13 branches.. (add opp points)
